chef/cookbooks/provisioner/templates/default/zzz-prompt.sh.erb

```diff
@@ -4,28 +4,34 @@ else
     prompt_suffix=">"
 fi
 
-case $(readlink /proc/$$/exe 2>/dev/null) in
-    */zsh)
-        PS1=$'<%= @zsh_prompt_from_template.call %> '
-        ;;
+case "$TERM" in
+    xterm*|vte*|linux*)
```

Comment:
There are plenty of non-dumb terminals which don't match that pattern, e.g. `screen*`, `rxvt*`.  Better to do terminal capability tests via `tput` or similar.